src/styles/Footer.less

```diff
@@ -0,0 +1,22 @@
+.footer {
+    height: 3rem;
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    background-color: #8b51c6;
```

Comment:
Use color variables in separate file?

Reply:
Legger til color.less nå slik det er gjort i repoet til rodekors.